runtime/bref/0.2/serverless.yaml

```
@@ -0,0 +1,52 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: my-symfony-app
+configValidationMode: error
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: prod
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
+        APP_RUNTIME: Runtime\Bref\Runtime
+        BREF_LOOP_MAX: 1 # Increase to keep the Lambda process alive between requests
+        # SYMFONY_DECRYPTION_SECRET: ${ssm:/stripe-notifications/prod-decrypt-private-key}
```

```suggestion
        # SYMFONY_DECRYPTION_SECRET: ${ssm:/my-symfony-app/prod-decrypt-private-key}
```

to be consistent with the name of the service